### PR TITLE
Adding data version to the data-manager pipeline

### PR DESCRIPTION
**Pull request by Zaknarfen**
- Added a data-manifest.json file with the data version
	as a property
	- Changed the import data script to not fetch the data
	version from the Agora app
	- Added logic to fetch the data version from the
	data-manifest.json

### CODE REVIEW

File: import-data.sh [hunk 1]
```diff
@@ -19,12 +19,9 @@ WORKING_DIR=$TMP_DIR/work
...
-wget https://raw.githubusercontent.com/Sage-Bionetworks/Agora/$TRAVIS_BRANCH/package.json -O $WORKING_DIR/package-$TRAVIS_BRANCH.json
-
 # Version key/value should be on his own line
-DATA_VERSION=$(cat $WORKING_DIR/package-$TRAVIS_BRANCH.json | grep data-version | head -1 | awk -F: '{ print $2 }' | sed 's/[",]//g' | tr -d '[[:space:]]')
-echo "package-$TRAVIS_BRANCH.json DATA_VERSION = $DATA_VERSION"
+DATA_VERSION=$(cat data-manifest.json | grep data-version | head -1 | awk -F: '{ print $2 }' | sed 's/[",]//g' | tr -d '[[:space:]]')
+echo "$TRAVIS_BRANCH branch, DATA_VERSION = $DATA_VERSION"
 
 # get data from synapse
 synapse -u $SYNAPSE_USERNAME -p $SYNAPSE_PASSWORD cat --version $DATA_VERSION syn18387112 | tail -n +2 | while IFS=, read -r id version; do
```
Comment (zaro0508) [inline import-data.sh]: I thought we were going to parameterize the synapse entity id as well?  Maybe you want to add that to the data-manifest.json?

Wondering if the synapse entity for team images should be added to data-manifest as well?

**Review by zaro0508 — CHANGES_REQUESTED** [review]

File: import-data.sh (hunk 1 above)
Comment (Zaknarfen) [reply import-data.sh]: Sure, makes sense I can add them as new properties in the `data-manifest.json`. I'll also change the id to the live data one.

**Review by zaro0508 — CHANGES_REQUESTED** [review]
The code change looks good.  Just noticed that the documentation in README no longer matches this new setup. Could you please update the doc and remove `Sage-Bionetworks/Agora` cloud from the image?  We no longer go to Agora repo to get the data version.

File: README.md [hunk 2]
```diff
@@ -13,8 +13,7 @@ self service update.
 # Worflow
 
 To deploy an updated data version to the Agora development database
-1. Increment `data-version` in [package.json](https://github.com/Sage-Bionetworks/Agora/blob/develop/package.json)
-on the `develop` branch.
+1. Increment `data-version` in `data-manifest.json` on the `develop` branch.
 2. Commit the change
 3. Rerun the [travis build](https://travis-ci.org/Sage-Bionetworks/agora-data-manager/builds)
```
Comment (zaro0508) [inline README.md]: re-run not needed anymore so maybe replace step 3 with "The [CI system](https://travis-ci.org/Sage-Bionetworks/agora-data-manager) automatically updates the new version to the DB. 

Probably can also remove "2. Rerun the agora-data-manager build for those branches." below as well.

**Review by zaro0508 — CHANGES_REQUESTED** [review]
slight update to docs, otherwise looks good.

**Review by zaro0508 — APPROVED** [review]

**Pull request merged**